packages/image_picker/image_picker/ios/Classes/FLTImagePickerPlugin.m

```diff
@@ -457,7 +461,7 @@ - (void)imagePickerController:(UIImagePickerController *)picker
       image = [FLTImagePickerImageUtil scaledImage:image
                                           maxWidth:maxWidth
                                          maxHeight:maxHeight
-                               isMetadataAvailable:originalAsset != nil];
+                               isMetadataAvailable:YES];
```

Comment:
What is this change for? It doesn't seem obviously related.

Reply:
Yes, this is not related to this PR but since it is just a small fix I wanted to add it here. This change is related to [#3873](https://github.com/flutter/plugins/pull/3873). `isMetadataAvailable` parameter is used to handle rotation problem for limited access to the photo library and it should be `YES` for all other occasions.

Reply:
If it's not related, then it would be much better to create a new PR for it. Having behavior-changing fixes hidden in unrelated PRs makes things much more difficult when doing things like trying to find the reason for certain changes with blame, or looking for causes of regressions.

It will also need tests, which would require adding even more (unrelated) code here.